<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FnTitleBar</class>
 <widget class="QWidget" name="FnTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>29</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>29</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>29</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
   
	background-color: rgba(255, 255, 255, 0);
    border::none;
}
#btnClose::hover
{
  
	background-color: rgb(215, 12, 38);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>32</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wgtMenu" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QMenuBar {
	background:transparent;

}

QMenuBar::item {

	
	
	/* 背景颜色 */
	background:transparent;
   
	color: rgb(255, 255, 255);
  
	

}
QMenuBar::item::hover {

	
	
	/* 背景颜色 */
	
	background-color: rgba(255, 255, 255, 30);
   
	color: rgb(255, 255, 255);
  
	

}
QMenu
{
   
	background-color: rgb(37, 37, 37);
    
	color: rgb(255, 255, 255);
}
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblTitle
{
  color:rgb(255,255,255);
  margin-left:64px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>726</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="btnMin">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnMax">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnClose">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
